rpc/lib/client/ws_client.go

```diff
@@ -60,10 +56,16 @@ type WSClient struct {
 	sentLastPingAt time.Time
 	reconnecting   bool
 
-	// Time allowed to read the next pong message from the server.
-	pongWait time.Duration
+	// Maximum reconnect attempts (0 or greater; default: 25).
+	maxReconnectAttempts int
+
+	// Time allowed to write a message to the server. 0 means block until operation succeeds.
+	writeWait time.Duration
+
+	// Time allowed to read the next message from the server. 0 means block until operation succeeds.
+	readWait time.Duration
 
-	// Send pings to server with this period. Must be less than pongWait.
+	// Send pings to server with this period. Must be less than readWait. If 0, no pings will be sent.
```

Comment:
do we enforce this anywhere? Or just up to user? If later we should document in a place where users will see.